public/google9826929a5028a0d1.html

```diff
@@ -0,0 +1 @@
+google-site-verification: google9826929a5028a0d1.html
```

Comment:
I added this for step 3 here: https://support.google.com/webmasters/answer/183668?hl=en
It allows Google to verify that I am an administrator for this site and use their search console tools. It's not really ideal but I feel like it's better than just hoping it works...

Reply:
Ah, yes! Excellent. I've used this in the past.

I _think_ they also let you set a DNS record, but… that's out of our hands.